### DISCUSSION TITLE
Extracting Boxes, scores, class prediction and ID

### DISCUSSION THREAD

**Discussion by Aminekabene** [mikel-brostrom/yolo_tracking#516] [Q&A]
Hi, I am trying to extract and store all of the Boxes, scores, class prediction and IDs  of all the objects detected in a single frame.

Is there a predefined function that returns all of the above information?

**Comment by mikel-brostrom** [comment]
> Is there a predefined function that returns all of the above information?

No. Here is what is stored in the saved `.txt` file:

https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/6f9f294914ad503a1efc850e8d8dddfb46ef146a/track.py#L229

you can replace all the -1 values for whatever you want, as these are placeholders for other information
